posthog/cdp/templates/rudderstack/template_rudderstack.py

```diff
@@ -0,0 +1,118 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="alpha",
+    id="template-rudderstack",
+    name="Send data to RudderStack",
+    description="Send data to RudderStack",
+    icon_url="/static/services/rudderstack.png",
+    hog="""
+let host := inputs.host
+let token := inputs.token
+
+let rudderPayload := {
+    'context': {
+        'app': {
+            'name': 'PostHogPlugin',
+        },
+        'os': {
+            'name': event.properties.$os
+        },
+        'browser': event.properties.$browser,
+        'browser_version': event.properties.$browser_version,
+        'page': {
+            'host': event.properties.$host,
+            'url': event.properties.$current_url,
+            'path': event.properties.$pathname,
+            'referrer': event.properties.$referrer,
+            'initial_referrer': event.properties.$initial_referrer,
+            'referring_domain': event.properties.$referring_domain,
+            'initial_referring_domain': event.properties.$initial_referring_domain,
+        },
+        'screen': {
+            'height': event.properties.$screen_height,
+            'width': event.properties.$screen_width,
+        },
+        'library': {
+            'name': event.properties.$lib,
+            'version': event.properties.$lib_version,
+        },
+        'ip': event.$ip,
+        'active_feature_flags': event.properties.$active_feature_flags,
+        'token': event.properties.token
+    },
+    'channel': 's2s',
+    'messageId': event.uuid,
+    'originalTimestamp': event.timestamp,
+    'userId': event.properties.$user_id ?? event.distinct_id,
```

Comment:
I think we need to make this configurable. 

For migrating people from the old code we would use this, but for new users `{person.uuid}` would be a way more stable choice.